src/createServer.js

```diff
@@ -1,8 +1,51 @@
+/* eslint-disable no-console */
 'use strict';
 
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
+const PORT = require('./PORT');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  return http.createServer((request, response) => {
+    response.setHeader('Content-type', 'text/plain');
+
+    if (request.url === '/favicon.ico') {
+      return;
+    }
+
+    if (request.url.includes('..')) {
+      response.statusCode = 404;
+      response.end('/../ not allowed');
+      // I have no idea how to pass this test:
+      // 'should return 400 for traversal paths'
+      // maybe axios normalizes the path and prevents traversal paths?
+    }
+
+    if (!request.url.startsWith('/file/')) {
+      response.statusCode = 200;
```

Comment:
That's ok. There is no error and the response is correct, so using `200` code is acceptable